include/zephyr/bluetooth/hci.h

```diff
@@ -131,6 +150,15 @@ int bt_hci_get_adv_handle(const struct bt_le_ext_adv *adv, uint8_t *adv_handle);
  */
 int bt_hci_get_adv_sync_handle(const struct bt_le_per_adv_sync *sync, uint16_t *sync_handle);
 
+/** @brief Get periodic advertising sync given an periodic advertising sync handle.
+ *
+ * @param handle The periodic sync set handle
+ *
+ * @retval The corresponding periodic advertising sync set object on success,
+ *         NULL if it does not exist.
+ */
+struct bt_le_per_adv_sync *bt_hci_get_per_adv_sync(uint16_t handle);
```

Comment:
I think the `lookup_handle` suffix works fine
```suggestion
struct bt_le_per_adv_sync *bt_hci_per_adv_sync_lookup_handle(uint16_t handle);
```